backend/api/admin.py

```
@@ -1,3 +1,41 @@
+        ('Important dates', {'fields': ('last_login', 'date_joined')}),
+    )
+    add_fieldsets = (
+        (None, {
+            'classes': ('wide',),
+            'fields': ('firebase_uid', 'email', 'password1', 'password2', 'role'),
+        }),
+    )
+    search_fields = ('email', 'first_name', 'last_name')
+    ordering = ('email',)
+
+    def get_inlines(self, request, obj=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_inlines`

(ANN201)